Source/WebCore/page/PerformanceEventTiming.h

```diff
@@ -26,35 +26,29 @@
 
 #pragma once
 
-#include <WebCore/DOMHighResTimeStamp.h>
-#include <WebCore/EventNames.h>
-#include <WebCore/EventTarget.h>
-#include <WebCore/PerformanceEntry.h>
+#include "DOMHighResTimeStamp.h"
+#include "EventNames.h"
+#include "EventTarget.h"
+#include "PerformanceEntry.h"
+#include "PerformanceEventTimingCandidate.h"
```

Comment:
I don't think we need to include this header. Just upward declare `PerformanceEventTimingCandidate`.